common/saber_base.h

```diff
@@ -36,6 +36,8 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(POWERSAVE)                    \
     DEFINE_EFFECT(BATTERY_LEVEL)                \
     DEFINE_EFFECT(FAST_ON)                      \
+    DEFINE_EFFECT(SWAP)            		\
```

Comment:
Why not use USER1 instead of SWAP?

Reply:
Didn't even see that , and was overthinking the sound vs blade effects...it works very easily now. Thanks.